neuromation/cli/formatters/ftable.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass
+from enum import Enum
+from itertools import zip_longest
+from textwrap import wrap
+from typing import Any, Iterator, List, Optional, Sequence
+
+
+__all__ = ["table"]
+__version__ = "0.1"
+
+
+@dataclass(frozen=True)
+class ColumnWidth:
+    min: Optional[int] = None
+    max: Optional[int] = None
+    width: Optional[int] = None
+
+    def __post_init__(self) -> None:
+        if self.width and self.width < 0:
+            raise ValueError("Width must be positive integer")
```

Comment:
But `0` also passes this check.

Reply:
It's ok. `0` and `None` is "unspecified" for me